apps/odk_publish/forms.py

```diff
@@ -162,4 +174,41 @@ def clean(self):
                         )
                     }
                 )
+            self.file_data = data
+        return self.cleaned_data
+
+
+class AppUserConfirmImportForm(import_export_forms.ConfirmImportForm):
```

Comment:
Maybe this could use the FileFormatChoiceField to clean and load the input format instance:

```suggestion
class AppUserConfirmImportForm(import_export_forms.ConfirmImportForm):
    format = FileFormatChoiceField(widget=forms.HiddenInput)
```